tncc.py

```diff
@@ -272,6 +276,9 @@ def __init__(self, vpn_host, device_id=None, funk=None, platform=None, hostname=
 
         self.deviceid = device_id
 
+        self.setup_mechanize()
+	
+    def setup_mechanize(self):
```

Comment:
Could this method be renamed to better reflect that it doesn't only setup headless browser, but also does the host checker result submission?